python/awaazde/api_client.py

```diff
@@ -54,6 +54,8 @@ def _request(self, method, url, **kwargs):
             s = requests.Session()
             retries = Retry(total=5, backoff_factor=1, status_forcelist=[502, 503, 504], method_whitelist=False)
             s.mount('https://api.awaaz.de/', HTTPAdapter(max_retries=retries))
+            s.headers.update({'referer':'https://app.awaaz.de/'})
```

Comment:
@poojalanghnoda Please add a comment specifying the reason behind adding the header and cookie.